qa/restart-upgrade/build.gradle

```diff
@@ -19,40 +20,86 @@ testClusters {
         plugin(project.tasks.zipBwcPlugin.archiveFile)
         setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
         setting 'http.content_type.required', 'true'
-        environment "LD_LIBRARY_PATH", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/knnlib"
-        systemProperty "java.library.path", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/knnlib"
+        environment "LD_LIBRARY_PATH", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/knnlib;${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/lib"
```

Comment:
This will fail for 1.0 and 1.1 and 1.2?

Reply:
Do you mean when the version is upgraded to 1.0, 1.1 or 1.2?
Or upgraded from these versions to latest OpenSearch version?

Reply:
Upgrade from

Reply:
no, they don't fail. We can verify it from the checks. The checks cover all those scenarios and all the checks have passed.

Reply:
Oh I see so you set the location in both places. I think that makes sense.